app/vmui/packages/vmui/src/components/ExploreAlerts/BaseAlert/index.tsx

```diff
@@ -0,0 +1,89 @@
+import "./style.scss";
+import { Alert as APIAlert } from "../../../types";
+import { createSearchParams } from "react-router-dom";
+import Button from "../../Main/Button/Button";
+import Badges from "../Badges";
+import {
+  SearchIcon,
+} from "../../Main/Icons";
+import dayjs from "dayjs";
+
+interface BaseAlertProps {
+  item: APIAlert;
+}
+
+const BaseAlert = ({ item }: BaseAlertProps) => {
+  const query = item?.expression;
+
+  const openQueryLink = () => {
+    const params = {
+      "g0.expr": query,
+      "g0.end_time": ""
+    };
+    window.open(`#/?${createSearchParams(params).toString()}`, "_blank", "noopener noreferrer");
+  };
+
+  return (
+    <div className="vm-explore-alerts-alert-item">
+      <table>
+        <tbody>
+          <tr>
+            <td colSpan={2}>
+              <Button
+                size="small"
+                variant="outlined"
+                color="gray"
+                startIcon={<SearchIcon />}
+                onClick={openQueryLink}
+              >
+                <span className="vm-button-text">Show query</span>
```

Comment:
```suggestion
                <span className="vm-button-text">Run query</span>
```

When built locally, this button seems out of place:
<img width="612" height="172" alt="image" src="https://github.com/user-attachments/assets/c7448a92-42c1-478b-a8fd-27827476ee73" />

Can we move it to the right border of the whole block, below the FIRING/PENDING icons?